core/src/toga/app.py

```diff
@@ -516,13 +517,39 @@ def exit_full_screen(self):
             self._impl.exit_full_screen(self._full_screen_windows)
             self._full_screen_windows = None
 
+    @property
+    def cursor_position(self):
+        """Return the cursor position with respect to the whole app."""
+        return self._impl.get_cursor_position()
+
+    @cursor_position.setter
+    def cursor_position(self, value: Tuple[int, int]):
+        """Set the cursor position with respect to the whole app."""
+        if not isinstance(value, tuple) or len(value) != 2:
+            print("Error! Invalid cursor position value or type.")
+            return
```

Comment:
If inputs are invalid, they should raise an ValueError, not print a message. Also - the error message should provide some indication of what "correct" is, not just say "invalid".
```suggestion
            raise ValueError("Cursor position must be an (x,y) tuple")
```